src/content/deployment/flavors.md

```diff
@@ -52,32 +58,63 @@ Open your project in Xcode.
 </li>
 <li>
 
-Select **Product** > **Scheme** > **New Scheme** from the menu to
-add a new `Scheme`.
+Navigate to the **Build Settings** of the target(s)
+and follow these instructions:
+
+</li>
+
+If `DART_DEFINES` doesn't exist yet:
+
+* Click the **+** button in the top left.
+* Select **Add User-Defined**.
+* Enter `DART_DEFINES` as the name.
+* Enter the **base64 encoded version**
+  of `FLUTTER_APP_FLAVOR=<value>` as the value,
+  where `value` is the name of the flavor
+  that is expected to be the `appFlavor` in Dart.
+
+:::note Important
+You must encode the entire string as `base64`.
```

Comment:
If I want my app flavor to be `foo`, should I set `DART_DEFINES` to `FLUTTER_APP_FLAVOR=Zm9v` (only `foo` is encoded) or to `RkxVVFRFUl9BUFBfRkxBVk9SPWZvbw==` (all of `FLUTTER_APP_FLAVOR=foo` is encoded)?

I believe it's the latter, perhaps an example would help clarify?

```suggestion
You must encode the entire string as `base64`.

For example, for flavor `foo` set `DART_DEFINES` to `RkxVVFRFUl9BUFBfRkxBVk9SPWZvbw==` (base64 encoding of `FLUTTER_APP_FLAVOR=foo`).
```